toolkit/scss/forms/_combinations.scss

```diff
@@ -13,10 +13,6 @@ $border-width: $gutter / 6;
   padding-top: $gutter-two-thirds;
   margin: -$gutter 0 0 0;
 
-  &:not(:last-of-type) {
-    padding-bottom: $gutter;
-  }
-
```

Comment:
That's pretty weird that the grey border isn't continuous and suggests a markup problem.  Also this isn't compatible with ie8 which we still support, so we should be finding an alternative anyway.